crates/metrics/src/txpool_metrics.rs

```diff
@@ -1,24 +1,94 @@
-use crate::global_registry;
-use prometheus_client::metrics::histogram::Histogram;
+use crate::{
+    buckets::{
+        buckets,
+        Buckets,
+    },
+    global_registry,
+};
+use prometheus_client::metrics::{
+    gauge::Gauge,
+    histogram::Histogram,
+};
 use std::sync::OnceLock;
 
 pub struct TxPoolMetrics {
-    pub tx_size_histogram: Histogram,
+    /// Size of transactions in the txpool in bytes
+    pub tx_size: Histogram,
+    pub number_of_transactions: Gauge,
+    pub number_of_transactions_pending_verification: Gauge,
+    pub number_of_executable_transactions: Gauge,
```

Comment:
Should these have doc comments as well?

Reply:
yes: 9cee8696405e3aafcfcd7de4bb14fa8c976c86cc